src/service/Service.ts

```
@@ -63,7 +76,7 @@ export default class Service {
             // 토큰 만료시 refresh
             const currentTime = new Date().getTime();
             const timeDifference = currentTime - this.tokenGenerateTime;
-            const expiresIn = this.cookies.get("expires_in")?.value;
+            const expiresIn = this.cookies.expires_in;
 
             if (timeDifference >= Number(expiresIn) * 1000) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)